src/expr.rs

```diff
@@ -1260,17 +1292,27 @@ fn rewrite_call_inner<R>(context: &RewriteContext,
     } else {
         offset
     };
-    let inner_context = &RewriteContext { block_indent: block_indent, ..*context };
 
-    let items = itemize_list(context.codemap,
-                             args.iter(),
-                             ")",
-                             |item| item.span.lo,
-                             |item| item.span.hi,
-                             |item| item.rewrite(&inner_context, remaining_width, offset),
-                             span.lo,
-                             span.hi);
-    let mut item_vec: Vec<_> = items.collect();
+
+    let mut item_vec: Vec<_> = {
+        let mut inner_context = RewriteContext {
```

Comment:
This seems like a common construct. Could we extract this to a method or macro somehow?